apps/web/vibes/soul/primitives/alert/index.tsx

```diff
@@ -1,11 +1,12 @@
 import { clsx } from 'clsx';
 import { X } from 'lucide-react';
+import { ReactNode } from 'react';
 
 import { Button } from '@/vibes/soul/primitives/button';
 
 interface Props {
   variant: 'success' | 'warning' | 'error' | 'info';
-  message: string;
+  message: ReactNode;
```

Comment:
Changed to `ReactNode` because we want to be able to toast messages that include `Link` components.